kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/net/NetworkAdminService.java

```diff
@@ -66,7 +66,10 @@ public List<? extends NetInterfaceConfig<? extends NetInterfaceAddressConfig>> g
      * @param mtu
      *            - required MTU for the interface, -1 to keep the automatic default
      * @throws KuraException
+     * 
+     * @deprecated Since 2.4
```

Comment:
Do we suggest the use of other APIs in replacement of these?

Reply:
It doesn't exist a 1-to-1 replacement of this API. We can suggest to use directly the ConfigurationService.